nix/ucm.nix

```diff
@@ -111,13 +111,6 @@ in
       runHook postInstall
     '';
 
-    postInstall = ''
-      installShellCompletion --cmd ucm \
-        --bash <(${unison} --bash-completion-script ${unison}) \
-        --fish <(${unison} --fish-completion-script ${unison}) \
-        --zsh <(${unison} --zsh-completion-script ${unison})
-    '';
```

Comment:
Can you open an issue re: restoring completion? You can assign it to me, and link to this PR or diff … and to #97.

Reply:
Actually, I‘ll just do this.